test/modules/packages/ZMQ/interop-py/wrapper.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+pyscript=$1
+
+if [[ -f pyzmq-venv/bin/activate ]] ; then
+    source pyzmq-venv/bin/activate
+    python $pyscript
+    deactivate
```

Comment:
What happens if this `deactivate` doesn't get called due to something going wrong? Will the parent environment be impacted?